setup.py

```diff
@@ -54,10 +54,7 @@
             "google-cloud-bigquery[pandas]",
             "google-cloud-bigquery-storage",
         ],
-        "databricks": [
-            "databricks-sql-connector",
-            "databricks-cli",
-        ],
+        "databricks": ["databricks-sql-connector", "databricks-connect"],
```

Comment:
@eakmanrq I removed `databricks-cli` because its [no longer supported by Databricks](https://docs.databricks.com/en/dev-tools/cli/index.html#information-for-legacy-databricks-cli-users) and I needed the new version to use from the utility scripts.

Also, do you see an issue adding `databricks-connect` here? It was required by the integration tests because they deal with DataFrame's

Reply:
You can't include `databricks-connect`. The reason is that the user needs to use a specific version of `databricks-connect` based on their databricks runtime. For example if they are targetting a cluster with runtime 13.3 then they need to use `databricks-connect==13.3.*`

Reply:
Thanks, I thought something like that would be the case. I've shifted the installation of `databricks-connect` to the `install-prerequisites.sh` script that CircleCI runs before triggering tests